README.md

```diff
@@ -38,23 +38,22 @@ It is part of the larger Asynchronous Network Managment System (ANMS) managed fo
        `'(_ )_)(_)_)'
 
 
-This tool uses the JSON representation of an Application Data Model (ADM) to
+This tool uses the YANG representation of an Application Data Model (ADM) to
```

Comment:
Technically the version of ACE used here is still ingesting JSON ADMs, but... that will be changing in the fy24 branch anyway so not a big deal if you want to leave this for now.